models.py

```diff
@@ -960,6 +960,7 @@ def is_accessible(self):
             self.can_create = True
             return True
         if current_user.has_role(Role.ROLE_FINANCE) or current_user.has_role(Role.ROLE_AUTHORIZER):
+            self.can_create = False
```

Comment:
This will prevent the FINANCE and AUTHORIZER from creating proposals. It looks like because we included 
```
self.can_create = True
```
to the other role, its overwriting the main one for this View.